### ISSUE TITLE
Verify taxa: split info$outdated_taxa in two df

### ISSUE THREAD

**Issue by peterdesmet** [trias-project/trias#28]
I like to list the outdated synonyms and outdated unmatched taxa separately:

```R
verification$info$outdated_taxa %>% filter(is.na(bb_key)) %>% nrow()
verification$info$outdated_taxa %>% filter(!is.na(bb_acceptedKey)) %>% nrow()
```

That however returns an error if `verification$info$outdated_taxa` is NULL (often the case). I would therefore prefer two dataframes `outdated_unmatched_taxa` and `outdated_synonyms`:

```R
verification$info$outdated_unmatched_taxa %>% nrow()
verification$info$outdated_synonyms %>% nrow()
```

**Comment by damianooldoni** [comment]
Yes, It makes sense. 
I would extend your suggestion to all elements of list `info´.

**Comment by peterdesmet** [comment]
I don’t think it is necessary for all elements
of $info.

**Comment by peterdesmet** [comment]
You mean returning empty df instead? 👌Ok, as long as I can use it easily.

**Comment by damianooldoni** [comment]
Yes, $info as a list of dataframes (tibbles), where some of them can have zero rows.

**Comment by damianooldoni** [comment]
This issue can be closed. Issue solved by PR #32
